dbms/src/Storages/StorageDeltaMerge.cpp

```diff
@@ -95,6 +97,12 @@ StorageDeltaMerge::StorageDeltaMerge(
         is_common_handle = tidb_table_info.is_common_handle;
         pk_is_handle = tidb_table_info.pk_is_handle;
     }
+    else
+    {
+        const auto mock_table_id = MockTiDB::instance().newTableID();
```

Comment:
It seems we cannot restore the table created through ch client after restart tiflash.

Reply:
Well
* We don't have any crash test cases in the mock test now, and no plan to add in the near future
* Even in mock test, `DBGInvoke mock_tidb_table` is the recommended way to create a table
* Those test cases using `create table ... ` through ch-client should be rewritten in other forms IMO, but I don't get time to do that. This is a workaround other than skipping those cases